rebar.config

```diff
@@ -1,8 +1,12 @@
-{require_otp_vsn, "R15|R16|17"}.
+{require_otp_vsn, "R15|R16|17|18"}.
 
 {cover_enabled, true}.
 {eunit_opts, [verbose]}.
-{erl_opts, [warnings_as_errors, debug_info, nowarn_deprecated_type]}.
```

Comment:
OTP 18 produces warnings about deprecated `erlang:now/0` and investigating if it's safe to replace them was not in the scope of my work.